.cirlceci: Setup nvidia runtime for cu as well

seemethere opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#41268 .cirlceci: Setup nvidia runtime for cu as well**

We also want nvidia runtime packages to get installed when the
BUILD_ENVIRONMENT also includes "*cu*"

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Differential Revision: [D22505885](https://our.internmc.facebook.com/intern/diff/D22505885)

seemethere committed:
.cirlceci: Setup nvidia runtime for cu as well

We also want nvidia runtime packages to get installed when the
BUILD_ENVIRONMENT also includes "*cu*"

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

[ghstack-poisoned]

malfet commented on .circleci/scripts/setup_ci_environment.sh:
```diff
@@ -2,7 +2,7 @@
 
 # Set up NVIDIA docker repo
-if [[ "${BUILD_ENVIRONMENT}" == *cuda* ]]; then
+if [[ "${BUILD_ENVIRONMENT}" == *cuda* ]] || [[ "${BUILD_ENVIRONMENT}" == *cu* ]]; then
```
`*cu*` should cover both cases, shouldn't it?
```suggestion
if [[ "${BUILD_ENVIRONMENT}" == *cu* ]]; then
```

seemethere replied:
Changed to this, it'd probably good for us to look to actually just get rid of the `BUILD_ENVIRONMENT` variable, I feel like it causes more trouble then benefits

malfet commented on .circleci/scripts/setup_ci_environment.sh:
```diff
@@ -42,7 +42,7 @@ sudo apt-get -y remove linux-image-generic linux-headers-generic linux-generic d
 # presumably retrying at the single-package level via the
 # apt.conf.d/80-retries technique.
-if [[ "${BUILD_ENVIRONMENT}" == *cuda* ]]; then
+if [[ "${BUILD_ENVIRONMENT}" == *cuda* ]] || [[ "${BUILD_ENVIRONMENT}" == *cu* ]]; then
```
Same as above
```suggestion
if [[ "${BUILD_ENVIRONMENT}" == *cu* ]]; then
```

seemethere committed:
Update on ".cirlceci: Setup nvidia runtime for cu as well"


We also want nvidia runtime packages to get installed when the
BUILD_ENVIRONMENT also includes "*cu*"

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

[ghstack-poisoned]
